lib/ingestors/ingestor_event_rest.rb

```diff
@@ -826,10 +826,13 @@ def process_uu(url)
                 event.country = loc.last.strip
               when 'provider'
                 event.organizer = element.text
+              # ugly implementation so that TeSS does not shift timezone too much
               when 'startdate', 'courseDate'
-                event.start = element.text.to_s.to_time
+                event.start = element.text.to_s.split
+                event.start = event.start[0, event.start.length - 1].join(' ').to_s_to_time
```

Comment:
this code confuses me (and is repeated...) can you make a separate method and test it?